src/avgScrabbleScore.js

```diff
@@ -1,20 +1,68 @@
-let scrabbleScore = require('./scrabbleScore');
+ let scrabbleScore = require('./scrabbleScore');
 
 /**
  * Given an array of words, return the average scrabble
  * score.
  *
  * Use your implementation of scrabbleScore.
  */
+let SCRABBLE_TILE_SCORES = {
+  a: 1, b: 3, c: 3, d: 2, e: 1,  f: 4,
+  g: 2, h: 4, i: 1, j: 8, k: 5,  l: 1,
+  m: 3, n: 1, o: 1, p: 3, q: 10, r: 1,
+  s: 1, t: 1, u: 1, v: 4, w: 4,  x: 8,
+  y: 4, z: 10,
+};
+
+// function scrabbleScore(word) {
```

Comment:
Unless these comments are meant to communicate something to me, remove them before submitting. If they are meant to communicate something, say what.